docs/getting_started/index.rst

```diff
@@ -16,10 +16,10 @@ issue_, start an asynchronous discussion_, or go to Discord_ for synchronous
 interaction.
 
 The task of getting started with **xNVMe** will take you through
-:ref:`sec-building` with a companion section on :ref:`sec-building-toolchain`
+:ref:`sec-gs-building` with a companion section on :ref:`sec-building-toolchain`
```

Comment:
Should `sec-building-toolchain` be `sec-gs-building-toolchain`?

Reply:
Yeah, that would be sensible, but not the focus of this PR.